src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/ResendStaffNotificationsCoreCaseDataTest.java

```diff
@@ -0,0 +1,218 @@
+
+        EmailData emailData = emailDataArgument.getValue();
+        assertThat(emailData.getTo()).isEqualTo("recipient@example.com");
+        assertThat(emailData.getSubject()).isEqualTo("Claim " + claim.getReferenceNumber() + " issued");
+        assertThat(emailData.getMessage()).isEqualTo("Please find attached claim.");
+    }
+
+    @Test
+    public void shouldRespond409AndNotProceedForMoreTimeRequestedEventWhenMoreTimeNotRequested() throws Exception {
+        final String event = "more-time-requested";
+
+        givenForSearchForCitizen(CASE_REFERENCE);
+
+        makeRequest(CASE_REFERENCE, event).andExpect(status().isConflict());
+
+        verify(notificationClient, never()).sendEmail(any(), any(), any(), any());
+    }
+
+    @Test
+    public void shouldRespond200AndSendNotificationsForMoreTimeRequestedEvent() throws Exception {
+        final String event = "more-time-requested";
+
+        givenForSearchForCitizenAndDef(CASE_REFERENCE);
+
+        makeRequest(CASE_REFERENCE, event).andExpect(status().isOk());
+
+        verify(notificationClient).sendEmail(eq("staff-more-time-requested-template"),
+            eq("recipient@example.com"),
+            any(),
+            eq("more-time-requested-notification-to-staff-" + CASE_REFERENCE));
+    }
+
+    @Test
+    public void shouldRespond409AndNotProceedForResponseSubmittedEventWhenResponseNotSubmitted() throws Exception {
+        final String event = "response-submitted";
+
+        givenForSearchForCitizen(CASE_REFERENCE);
+
+        makeRequest(CASE_REFERENCE, event).andExpect(status().isConflict());
+
+        verify(emailService, never()).sendEmail(any(), any());
+    }
+
+    @Test
+    public void shouldRespond200AndSendNotificationsForResponseSubmittedEvent() throws Exception {
+        final String event = "response-submitted";
+
+        given(coreCaseDataApi.searchForCitizen(
+            eq(AUTHORISATION_TOKEN),
+            eq(SERVICE_TOKEN),
+            eq(USER_ID),
+            eq(JURISDICTION_ID),
+            eq(CASE_TYPE_ID),
+            eq(ImmutableMap.of("case.referenceNumber", CASE_REFERENCE, "page", PAGE))
+            )
+        ).willReturn(listOfCaseDetailsWithDefResponse());
+
+        makeRequest(CASE_REFERENCE, event).andExpect(status().isOk());
+
+        verify(emailService).sendEmail(eq("sender@example.com"), emailDataArgument.capture());
+
+        assertThat(emailDataArgument.getValue().getTo()).isEqualTo("recipient@example.com");
+        assertThat(emailDataArgument.getValue().getSubject())
+            .isEqualTo("Civil Money Claim defence submitted: John Rambo v John Smith " + CASE_REFERENCE);
+        assertThat(emailDataArgument.getValue().getMessage()).contains(
+            "The defendant has submitted a full defence which is attached as a PDF",
+            "Email: j.smith@example.com",
+            "Mobile number: 07873727165"
+        );
+    }
+
+    private void givenForSearchForCitizen(String caseReference) {
```

Comment:
please inline, extracting as a method just makes this harder to read and see what's happening